### ISSUE TITLE
Incorrect lower confidence interval with in sample predictions from AutoARIMA model

### ISSUE THREAD

**Issue by bjeffrey92** [Nixtla/statsforecast#150]
With an ARIMA model fitted with `AutoARIMA()`, the lower confidence intervals of the within sample predictions appear to be incorrect when multiple confidence intervals are requested via the `level` argument to `predict_in_sample()`. The order they are returned in is reversed so the lowest confidence interval has the greatest value. See the code below:

```
In [8]: import pandas as pd
   ...: from statsforecast.arima import AutoARIMA
   ...:
   ...: df = pd.read_csv("https://auto-arima-results.s3.amazonaws.com/M4-Hourly.csv")
   ...: data = df.y.tail(50).values
   ...:
   ...: model = AutoARIMA().fit(data)
   ...: in_sample_preds = model.predict_in_sample(level=(50, 80, 95))
   ...:
   ...: print(in_sample_preds.tail(5))
   ...: print(in_sample_preds.apply(lambda x: x - in_sample_preds["mean"]).tail(5))
   ...:
          lo_95%        lo_80%        lo_50%          mean        hi_50%        hi_80%        hi_95%
45  28095.930499  27738.005787  27338.012634  28493.610840  28891.291182  29249.215894  29649.209047
46  26553.488552  26195.563840  25795.570687  26951.168894  27348.849235  27706.773947  28106.767100
47  25438.876434  25080.951721  24680.958568  25836.556775  26234.237116  26592.161828  26992.154982
48  24794.698166  24436.773454  24036.780300  25192.378507  25590.058848  25947.983560  26347.976714
49  24839.403239  24481.478526  24081.485373  25237.083580  25634.763921  25992.688633  26392.681787
        lo_95%      lo_80%       lo_50%  mean      hi_50%      hi_80%       hi_95%
45 -397.680341 -755.605053 -1155.598207   0.0  397.680341  755.605053  1155.598207
46 -397.680341 -755.605053 -1155.598207   0.0  397.680341  755.605053  1155.598207
47 -397.680341 -755.605053 -1155.598207   0.0  397.680341  755.605053  1155.598207
48 -397.680341 -755.605053 -1155.598207   0.0  397.680341  755.605053  1155.598207
49 -397.680341 -755.605053 -1155.598207   0.0  397.680341  755.605053  1155.598207
```

I have traced the source of the bug back to [this line](https://github.com/Nixtla/statsforecast/blob/main/statsforecast/arima.py#L2454). I will submit a PR to fix

Note, I have checked and the same error does not apply to out of sample predictions

**Comment by kdgutier** [comment]
Hi @bjeffrey92,

I made comments to the [PR](https://github.com/Nixtla/statsforecast/pull/151).
As mentioned there we intend to keep the reverse order of the levels in the ARIMA, as it advantageous for probabilistic predictions wrangling. We think the way to work this will be to fix `predict_in_sample`. 

Can you please elaborate more on the problem?

**Comment by bjeffrey92** [comment]
Hi @kdgutier, 

I'm rather confused. The problem that I see is that the labelling of the lower confidence intervals are incorrect, in the example I showed above the lower 95% CI is labelled as lo_50% is it not?

The quantiles are computed on this line https://github.com/Nixtla/statsforecast/blob/main/statsforecast/arima.py#L2453 in one order and then labelled on the line below in the reverse order. 

The same thing does not occur for out of sample predictions (forecasts): https://github.com/Nixtla/statsforecast/blob/main/statsforecast/arima.py#L1433 where the labelling is applied in the same order as the quantiles are computed

**Comment by FedericoGarza** [comment]
Hi @bjeffrey92! Thank you very much for letting us know about this problem. We solved the issue [here](#165), which is now fixed in the `main` branch. The next release will include the fix on `PyPI` and `conda`. :D

**Comment by bjeffrey92** [comment]
Thanks @FedericoGarza!!
